docs/tutorials/semantic_search/conversation_search_with_byte_quantized_vector.md

```diff
@@ -0,0 +1,278 @@
+# Topic
+
+> This tutorial doesn't explain byte-quantized vectors in detail. For more information, see [Byte-quantized vectors in OpenSearch](https://opensearch.org/blog/byte-quantized-vectors-in-opensearch/).
+
+This tutorial shows how to build semantic search using the [Cohere Embed model](https://docs.cohere.com/reference/embed) and byte-quantized vectors in OpenSearch.
+
+The Cohere Embed v3 model supports several `embedding_types`. This tutorial uses the `int8` type for byte-quantized vectors. 
+
+Note: Replace the placeholders that start with `your_` with your own values.
```

Comment:
I would suggest that the your_xxx placeholders be emphasized somehow in the code samples, e.g., in italic or boldface or something.

Reply:
It will pollute the REST API sample request by doing this.  I think we should be good to leave as is.